.env.dev

```diff
@@ -1,11 +1,21 @@
+INSEE_CLIENT_ID=Nv5aanlGHjVunAwRm3VuxfqSM60a
+INSEE_CLIENT_SECRET=wnHB4OK26iz2wPfxJVZfcxtfDToa
+INSEE_CLIENT_ID_FALLBACK=CS0pQtZoh1YR2KDQxn0CaIgA35ca
+INSEE_CLIENT_SECRET_FALLBACK=Ur2XB1IOcftoSIoj64mphVZKWdoa
+NODE_ENV=production
 SENTRY_DSN=
-INSEE_CLIENT_ID=
-INSEE_CLIENT_SECRET=
-INSEE_CLIENT_ID_FALLBACK=
-INSEE_CLIENT_SECRET_FALLBACK=
-INPI_LOGIN=
-INPI_PASSWORD=
-INPI_SITE_LOGIN=
-INPI_SITE_PASSWORD=
+VITE_SENTRY_FRONT_DSN=
 MATOMO_SITE_ID=
-UPTIME_ROBOT_API_KEY=
\ No newline at end of file
+MATTERMOST_HOOK=
+UPTIME_ROBOT_API_KEY=ur1272389-66032dec6571197734c275c9
+ALTERNATIVE_SEARCH_ROUTE=
+REDIS_URL=redis://127.0.0.1:6379
+INSEE_DIRIGEANT_CLIENT_ID=annuaire-entreprises
+INSEE_DIRIGEANT_CLIENT_SECRET=d123c1d3-9e44-4703-91b3-68cff2ba3af2
+PROXY_API_KEY=pjR8DuKb4U6@NL4fBoEu9nTU
+SITE_URL=http://localhost:3000
+MONCOMPTEPRO_CLIENT_ID=233ba6a6fd9303ab727159eeed704bcbc2549e56cd294fdf58621f01b1856898d18c6b20dd2cec21d4dc06746453331328a1b2c1abe365a92bb2a2f3b9d68c69
+MONCOMPTEPRO_CLIENT_SECRET=031da300506ac31fac6feaf88575b94b886eae7a68c64327ee03e0fa635ae9ae28572e4de53bb435b3fce4f8ac8484334d71de48f3c9abed3709c730173e0fe0
```

Comment:
It's that normal to commit an env key with `_SECRET`

Reply:
Definitely not ! Even if it is not prod client_id_/secret, this is an epic fail 😅